jsk_topic_tools/src/lightweight_throttle_nodelet.cpp

```diff
@@ -94,13 +105,21 @@ namespace jsk_topic_tools
       advertised_ = true;
       pub_ = pnh_.advertise(opts);
     }
+
     ros::Time now = ros::Time::now();
-    if ((now - latest_stamp_).toSec() > 1 / update_rate_) {
+
+    if (latest_stamp_ > now) {
```

Comment:
I see that latest_stamp_ and now are got by `ros::Time::now` not from header.
Looks good.